src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -1,16 +1,17 @@
 package core.basesyntax;
 
+import java.util.stream.Stream;
+
 public class FindNumbers {
 
-    /**
-     * <p>Реализуйте метод getAllNumbers(String text) который принимает строку, cодержащую буквы,
-     * целые числа и иные символы.</p>
-     * <p>Требуется: все числа, которые встречаются в строке, поместить в отдельный целочисленный
-     * массив, каждый элемент этого массива умножить на 2. Метод должен возвращать этот массив.</p>
-     * <p>Пример: если дана строка "data 48 call 9 read13 blank0a", то в массиве должны оказаться
-     * числа 92, 18, 26 и 0.</p>
-     */
     public int[] getAllNumbers(String text) {
-        return null;
+        String textNums = text.replaceAll("[^-?\\d+$]", " ").trim();
+        int [] stringToInt = Stream.of(textNums.split("\\s+"))
+                .mapToInt(Integer::parseInt)
```

Comment:
```suggestion
                .mapToInt(Integer::parseInt).map(number -> number * 2)
```
this way you don't need loop below